gnosis/safe-react
# TxList: Add info about the next tx to be executed

**What**:
Closes #2044.
Add information about the next tx to be executed in the tx-list, in queue title, and in the tooltip of the rocket button.


**Why**:
There are some users that get confused and try to execute a tx from the queue list. 


**How**:
I've made use of the hook `safeNonceSelector` in order to get the next nonce for the safe.


**How to test it**:
- Create a tx with a nonce bigger than the one suggested by the app.
- Check the queue title is `Transaction with nonce NN needs to be executed next.`
- Check that the Rocket button tooltip is `Transaction with nonce NN needs to be executed next.`
- Check the Execute button tooltip is `Transaction with nonce NN needs to be executed next.`

https://www.loom.com/share/44567800263342d98b584b84eb794bef


**Checklist**:
- [ ] Documentation added N/A
- [ ] Tests N/A
